cumulusci/tasks/release_notes/parser.py

```diff
@@ -283,3 +284,35 @@ def _add_issue_comment(self, issue):
                 break
         if not has_comment:
             issue.create_comment("{} {}".format(comment_prefix, version_str))
+
+
+class InstallLinkParser(BaseChangeNotesParser):
+    prod_url_template = (
+        "https://login.salesforce.com/packaging/installPackage.apexp?p0={}"
+    )
+
+    test_url_template = (
+        "https://test.salesforce.com/packaging/installPackage.apexp?p0={}"
+    )
+
+    def __init__(self, release_notes_generator, title):
+        super().__init__(title)
+        self.release_notes_generator = release_notes_generator
+
+    def parse(self, change_note):
+        # There's no need to parse lines, this parser gets its values from task options
+        pass
+
+    def render(self):
+        pv_id = self.release_notes_generator.version_id
+        if pv_id is not None:
+            pv_id = urllib.parse.quote_plus(pv_id)
+            return "\n\r".join(
```

Comment:
```suggestion
            return "\r\n".join(
```

(but why are we using Windows line endings anyway?)

Reply:
I honestly don't remember as these parsers were written years ago.  We could test not using Windows line endings and see if GitHub handles them the way we expect.